guides/common/modules/proc_using-freeipa-credentials-to-log-in-to-the-project-hammer-cli.adoc

```diff
@@ -1,60 +1,54 @@
 [id="Using_{FreeIPA-context}_credentials_to_log_in_to_the_{project-context}_Hammer_CLI_{context}"]
-= Using {FreeIPA} credentials to log in to the {Project} Hammer CLI
+= Logging in to Hammer CLI with {FreeIPA} credentials
 
-This section describes how to log in to your {Project} Hammer CLI with your {FreeIPA} (IdM) login and password.
+Authenticate to the {Project} Hammer CLI with your {FreeIPA} username and password.
 
 .Prerequisites
-* You have enrolled your {ProjectServer} into {FreeIPA} and configured it to use {FreeIPA} for authentication.
-More specifically, you have enabled access both to the {ProjectWebUI} and the {Project} API.
+* You have configured Hammer CLI to accept {FreeIPA} credentials.
+// See xref:configuring-hammer-cli-to-accept-{FreeIPA-context}-credentials_{context}[].
 ifndef::orcharhino[]
-For more information, see {InstallingServerDocURL}Using_FreeIPA_{project-context}[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
+For more information, see {InstallingServerDocURL}configuring-hammer-cli-to-accept-{FreeIPA-context}-credentials_{context}[Configuring Hammer CLI to accept {FreeIPA} credentials] in _{InstallingServerDocTitle}_.
 endif::[]
-* The host on which you run this procedure is configured to use {FreeIPA} credentials to log users in to your {Project} Hammer CLI.
-ifndef::orcharhino[]
-For more information, see {InstallingServerDocURL}Configuring_the_Hammer_CLI_to_Use_FreeIPA_User_Authentication_{project-context}[Configuring the Hammer CLI to Use {FreeIPA} User Authentication] in _{InstallingServerDocTitle}_.
-endif::[]
-* The host is an {FreeIPA} client.
-* An {FreeIPA} server is running and reachable by the host.
 
 .Procedure
-. Obtain a Kerberos ticket-granting ticket (TGT) on behalf of a {Project} user:
+. Authenticate as a user defined in {FreeIPA} to obtain a Kerberos ticket-granting ticket (TGT):
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-$ kinit idm_user
+$ kinit _{FreeIPA-context}_user_
 ----
 +
 [WARNING]
 ====
 If, when you were setting {FreeIPA} to be the authentication provider, you enabled access to both the {Project} API and the {ProjectWebUI}, an attacker can now obtain an API session after the user receives the Kerberos TGT.
```

Comment:
It's a bit difficult to read. Maybe change the order of things?
If you enabled access to both the {Project} API and the {ProjectWebUI}, when you were setting {FreeIPA} to be the authentication provider, an attacker...
OR 
end the first sentence with "authentication provider, be aware of a potential security risk. An attacker might obtain...

Reply:
Applied.